src/hawkbit-client.c

```diff
@@ -914,6 +914,17 @@ static gpointer download_thread(gpointer data)
         }
 
         g_mutex_lock(&active_action->mutex);
+        // skip installation if hawkBit asked us to do so
+        if (!artifact->do_install &&
+            (!artifact->do_maintenance || g_strcmp0(artifact->do_maintenance, "available") == 0)) {
+                active_action->state = ACTION_STATE_SUCCESS;
```

Comment:
Does this change anything, really?

Reply:
Good question. You mean that feedback alone would suffice?